### PR TITLE
Delete apps ported to the Portal

### PR DESCRIPTION
**Pull request by dnohales**
I haven't updated the `bundle-manifest-*.txt` files to remove the IDs, I'm not sure if I should do it. I also haven't updated any translation files

https://phabricator.endlessm.com/T24162

### CODE REVIEW

**Comment by wjt** [comment]
> I haven't updated the `bundle-manifest-*.txt` files to remove the IDs, I'm not sure if I should do it. 

Honestly, I don't know, but probably yes?

> I also haven't updated any translation files

I don't think this needs to be done by hand.

**Comment by dnohales** [comment]
Deleted entries for the mentioned files in the commit.

File: data/settings/icon-grid-C.json.in [hunk 1]
```diff
@@ -24,14 +24,9 @@
   "eos-folder-curiosity.directory": [
     "com.endlessnetwork.sciencesnacks.desktop",
     "com.endlessnetwork.shortfilms.desktop",
-    "com.endlessm.celebrities.en.desktop",
```
Comment (wjt) [inline data/settings/icon-grid-C.json.in]: Sorry, I should have been clearer in my comment. These icon grids are still used in the OS, so we should keep these app IDs in these files - we still want these apps to appear in these folders on the default desktop.

**Comment by dnohales** [comment]
@wjt Updated again.

**Comment by dbnicholson** [comment]
> I don't really know what the `bundle-manifest-*` files are used for, but it makes sense to me to remove these apps from them.

From `unzip_content.py`, `Generate bundle manifests for the image builder by personality`. I honestly don't even remember that and I'm sure I wrote the code to handle it. Nope, it was Roddy - https://github.com/endlessm/eos-image-builder/commit/88cbfdcaed9e3cdc06d6e7f55a7f45d06b869c4b. And then deleted by me in https://github.com/endlessm/eos-image-builder/commit/1d8ab78b7d0a002fe271655759817cee90c883a4. So, I'm very confident you can just nuke the files altogether. And if you're feeling frisky, delete the code from `unzip_content.py` that generates them (even though I'm pretty sure no one will ever run that script again).

**Comment by wjt** [comment]
@dbnicholson great to know. I've preserved that comment for posterity at https://phabricator.endlessm.com/T15338#788296

**Pull request merged**